deployment/ansible/roles/model-my-watershed.beaver/templates/spark-jobserver.conf.j2

```diff
@@ -1,4 +0,0 @@
-[/var/log/upstart/spark-jobserver.log]
-multiline_regex_before = ^\s+
-type: spark-jobserver
-tags: spark-jobserver,beaver
```

Comment:
Logs emitted from the service to `stdout` will go to `/var/log/upstart/{service-name}.log` by default. We can probably bring this file back and update the service name, type, and tags.